client/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+How to develop on this python client project 
+=
+<hr />
+
+Armada Project welcomes contributions from the opensource community.
+### Note that you need PYTHON 3+ to get this project working and this project does not work on native windows OS.
```

Comment:
```suggestion
This client is written primarily targeting supported versions of python 3 on Linux, Mac OS, and WSL on Windows. 
```

I suggest we make this more declarative, we can't list *all* the things we don't support so lets just list the things we do support.